setup.cfg

```diff
@@ -0,0 +1,111 @@
+[metadata]
+name = hvpy
+author = Helioviewer Project
+author_email = add@email.com
```

Comment:
@wafels 
Should we still use contact@helioviewer.org

Reply:
I added this for the time being.